examples/hotrod/services/config/config.go

```diff
@@ -59,4 +59,18 @@ var (
 
 	// RouteCalcDelayStdDev is standard deviation
 	RouteCalcDelayStdDev = RouteCalcDelay / 4
+
+	// Port settings, can be overwritten from command line
+
+	// Customer Service Port
+	CustomerPort = 8081
```

Comment:
Q: why do we need these in the config? The variables that hold the parsed flag values are already in the same package as the commands for services, so we can access the variables directly.